WinUIGallery/ControlPages/DesignGuidance/AccessibilityScreenReaderPage.xaml

```diff
@@ -460,6 +460,342 @@ AutomationProperties.Name="Contacts"&gt;
+            </core:ControlExample.Example>
+            <core:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;Grid&gt;
+    &lt;Grid.ColumnDefinitions&gt;
+        &lt;ColumnDefinition Width="200" /&gt;
+        &lt;ColumnDefinition Width="*" /&gt;
+        &lt;ColumnDefinition Width="200" /&gt;
+    &lt;/Grid.ColumnDefinitions&gt;
+
+    &lt;!--  The navigation pane for our app  --&gt;
+    &lt;StackPanel
+        Grid.Column="0"
+        Spacing="8"
+        Padding="6"
+        Background="{ThemeResource CardStrokeColorDefaultBrush}"
+        CornerRadius="{StaticResource ControlCornerRadius}"
+        AutomationProperties.LandmarkType="Navigation"&gt;
+        &lt;AutoSuggestBox
+            PlaceholderText="Search"
+            AutomationProperties.LandmarkType="Search"/&gt;
+        &lt;Button Content="Open settings" /&gt;
+    &lt;/StackPanel&gt;
+
+    &lt;!--  The main content of our app  --&gt;
+    &lt;StackPanel
+        Grid.Column="1"
+        Padding="6"
+        AutomationProperties.LandmarkType="Main"&gt;
+        &lt;TextBlock TextWrapping="WrapWholeWords"&gt;
+            Loren ipsum dolor sit anet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+            Ut enim ad minim venian, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea cannodo consequat.
+            Duis aute irure dolor in reprehenderit in voluptate velit esse cilium dolore eu fugiat nulla pariatur.
+            Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborun
+        &lt;/TextBlock&gt;
+    &lt;/StackPanel&gt;
+
+    &lt;!-- A custom sidebar with a custom landmark name --&gt;
+    &lt;StackPanel
+        Grid.Column="2"
+        Padding="6"
+        Background="{ThemeResource CardStrokeColorDefaultBrush}"
+        CornerRadius="{StaticResource ControlCornerRadius}"
+        Spacing="8"
+        AutomationProperties.LandmarkType="Custom"
+        AutomationProperties.LocalizedLandmarkType="Current viewers"&gt;
+        &lt;TextBlock
+            Text="Current viewers"
+            Style="{ThemeResource BodyStrongTextBlockStyle}"
+            AutomationProperties.HeadingLevel="Level1" /&gt;
+        &lt;TextBlock
+            Text="(No other users viewing)"
+            Style="{ThemeResource BodyTextBlockStyle}"
+            FontStyle="Italic" /&gt;
+    &lt;/StackPanel&gt;
+&lt;/Grid&gt;
+                    </x:String>
+            </core:ControlExample.Xaml>
+        </core:ControlExample>
+
+        <TextBlock
+            Margin="0,20,0,0"
+            Style="{ThemeResource SubtitleTextBlockStyle}"
+            AutomationProperties.HeadingLevel="3"
+            Text="Headings" />
+
+        <RichTextBlock>
+            <Paragraph>
+                <Bold>Headings</Bold> typically identify smaller groups of content.
+                They usually correspond to the "visual" headings in your UI—the text
```

Comment:
Space around the hyphen?

Reply:
*Technically* I don't believe you're supposed to include spaces around emdashes (see [Butterick's typography](https://practicaltypography.com/hyphens-and-dashes.html)), but I think, for clarity, it makes sense to add some small spaces.